sdks/python/apache_beam/runners/interactive/extensions/interactive_beam_side_panel/README.md

```diff
@@ -0,0 +1,111 @@
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+
+# interactive_beam_side_panel
+
+![Github Actions Status](https://github.com/apache/beam/workflows/Build/badge.svg)
+
+A side panel providing information of current interactive environment, data sets associated with PCollections, pipeline state, job status and source capture.
+
+
+This extension is composed of a Python package named `interactive_beam_side_panel`
+for the server extension and a NPM package named `interactive-beam-side-panel`
+for the frontend extension.
+
+
+## Requirements
+
+* JupyterLab >= 2.0
+
+## Install
+
+Note: You will need NodeJS to install the extension.
+
+```bash
+pip install interactive_beam_side_panel
+jupyter lab build
+```
+
+## Troubleshoot
+
+If you are seeing the frontend extension but it is not working, check
+that the server extension is enabled:
+
+```bash
+jupyter serverextension list
+```
+
+If the server extension is installed and enabled but you are not seeing
+the frontend, check the frontend is installed:
+
+```bash
+jupyter labextension list
+```
+
+If it is installed, try:
+
+```bash
+jupyter lab clean
+jupyter lab build
+```
+
+## Contributing
+
+### Install
```

Comment:
Related to @robertwb 's distribution story comment. How often we will release this? Who will release it? Releasing this will require a vote and a PMC approval, if you would like to add it to beam repo.

Reply:
The frontend extension should be published to the npm registry. The release process will be independent from the Python SDK release because they are not related.
There is a formal process and an automated service doing this, I can share you some link offline.